lib/Elastica/Query/Builder.php

```diff
@@ -1,885 +0,0 @@
-<?php
```

Comment:
It's hard to see it going but I agree its time to remove it, especially as it was not really maintained anymore.